x-pack/plugins/security_solution/public/common/components/hover_popover/hover_popover.tsx

```diff
@@ -5,52 +5,20 @@
  * 2.0.
  */
 
+import type { PopoverAnchorPosition } from '@elastic/eui';
 import { EuiPopover } from '@elastic/eui';
-import {
-  HOVER_ACTIONS_ALWAYS_SHOW_CLASS_NAME,
-  IS_DRAGGING_CLASS_NAME,
-} from '@kbn/securitysolution-t-grid';
-import React, { useCallback, useEffect, useMemo, useRef, useState } from 'react';
+import { IS_DRAGGING_CLASS_NAME } from '@kbn/securitysolution-t-grid';
+import type { PropsWithChildren } from 'react';
+import React, { useCallback, useEffect, useRef, useState } from 'react';
 
 /**
  * To avoid expensive changes to the DOM, delay showing the popover menu
  */
 const HOVER_INTENT_DELAY = 100; // ms
 
 interface Props {
-  /**
-   * Always show the hover menu contents (default: false)
-   */
-  alwaysShow?: boolean;
-  /**
-   * The hover menu is closed whenever this prop changes
-   */
-  closePopOverTrigger?: boolean;
-  /**
-   * The contents of the hover menu. It is highly recommended you wrap this
-   * content in a `div` with `position: absolute` to prevent it from effecting
-   * layout, and to adjust it's position via `top` and `left`.
-   */
-  hoverContent?: JSX.Element;
-  /**
-   * The content that will be wrapped with hover actions. In addition to
-   * rendering the `hoverContent` when the user hovers, this render prop
-   * passes `showHoverContent` to provide a signal that it is in the hover
-   * state.
-   */
-  render: (showHoverContent: boolean) => JSX.Element;
-
-  /**
-   * This optional callback is invoked when a close is requested via user
-   * intent, i.e. by clicking outside of the popover, moving the mouse
-   * away, or pressing Escape. It's only a "request" to close, because
-   * the hover menu will NOT be closed if `alwaysShow` is `true`.
-   *
-   * Use this callback when you're managing the state of `alwaysShow`
-   * (outside of this component), and you want to be informed of a user's
-   * intent to close the hover menu.
-   */
-  onCloseRequested?: () => void;
+  hoverContent: React.ReactNode;
```

Comment:
any reason for remove the documentation that was on this prop?

Reply:
Because the recommendation described no longer applies. 
This component is being used in only 2 places now (explained in the "Replacement" section of the PR description), none of them is setting `position: absolute` to the `hoverContent` and both work well, I could not reproduce the issue.

Maybe that was caused by `EuiTooltip` and they fixed that issue (it already sets `position: absolute`), or maybe because of the old way "hover actions" were implemented, I don't know, but since it's not reproducible anymore I decided to remove the doc, the prop name is intuitive enough.